autoblocks/_impl/testing/models.py

```diff
@@ -1,10 +1,28 @@
 import abc
 import dataclasses
 import functools
+import uuid
 from typing import Any
 from typing import Optional
 
 
+@dataclasses.dataclass()
```

Comment:
Note: This is temporarily here - I am going to create a tracer folder under _impl that contains its own models, but have held off to make the diff easier to review